src/main/fc/rc.c

```diff
@@ -827,46 +840,67 @@ static float quadraticBezier(float t, float p0, float p1, float p2) {
     return (1.0f - t) * (1.0f - t) * p0 + 2.0f * (1.0f - t) * t * p1 + t * t * p2;
 }
 
+// helper to solve a*t^2 + b*t + c = 0 for t in [0,1]
+static inline float solveQuadratic(float a, float b, float c) {
```

Comment:
Not in a terminal at the moment - so only have changes to look at here:

Perhaps call this function from within / or move code to `quadraticBezier()`